src/landing/RegisterForm.js

```diff
@@ -0,0 +1,117 @@
+import React, { useState } from 'react';
+import {
+  Button,
+  FormControl, FormHelperText, Input, InputLabel,
+} from '@material-ui/core';
+import {
+  validatePassword, validateUsername, validateFirstName, validateLastName, checkPasswordMatching,
+} from './Validations';
+import './styles.css';
+
+const RegisterForm = () => {
+  const [username, setUsername] = useState('');
+  const [password, setPassword] = useState('');
+  const [confirmPassword, setConfirmPassword] = useState('');
+  const [firstName, setFirstName] = useState('');
+  const [lastName, setLastName] = useState('');
+
+  const buttonDisabledValue = () => {
+    if (!validateUsername(username)
```

Comment:
You may write this as `return validateUsername(username) && validatePassword(password) &&...`

Reply:
fixed